firecrown/ccl/systematics/tests/test_pz.py

```diff
@@ -58,3 +59,29 @@ def test_photoz_shift_fill_nan():
     msk = z - 0.5 <= 0
     assert np.all(src.dndz_[msk] == 0)
     assert np.all(src.dndz_[~msk] != 0)
+
+
+def test_photoz_systematic_smoke():
```

Comment:
This test doesn't actually test anything. You are setting an attribute and then comparing it to what you set it to.